<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditManage</class>
 <widget class="QDialog" name="AuditManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>审核管理</string>
  </property>
  <widget class="QLineEdit" name="classEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>217</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>337</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>139</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>135</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>93</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>学号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>307</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>评优理由:(注:不超过200字)</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>611</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>908</x>
     <y>631</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>221</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>班级:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="SHSchkBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>三好生</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="XXGBchkBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>优秀学生干部</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="GQTYchkBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>优秀共青团员</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="genderEdit">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>174</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>178</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>性别:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
